projects/hslayers/src/components/layermanager/layermanager.service.ts

```diff
@@ -323,6 +327,11 @@ export class HsLayerManagerService {
     });
   }
 
+  refreshLists(): void {
```

Comment:
Can you add some comment why was cloning necessary, because I think we will forget otherwise?

Reply:
Good point, gonna do that.

Reply:
Reflected in 26ee014